ivy/container/experimental/statistical.py

```diff
@@ -324,3 +325,282 @@ def unravel_index(
         }
         """
         return self.static_unravel_index(self, shape, out=out)
+
+    @staticmethod
+    def static_quantile(
+        a: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        q: Union[ivy.Array, float],
+        /,
+        *,
+        axis: Optional[Union[Sequence[int], int]] = None,
+        keepdims: bool = False,
+        interpolation: str = 'linear',
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        out: Optional[ivy.Container] = None
+    ) -> ivy.Container:
+        """ivy.Container static method variant of ivy.quantile.
+        This method simply wraps the function, and so the docstring
+        for ivy.quantile also applies to this method with minimal
+        changes.
+
+    Parameters
```

Comment:
Needs aligning with the rest of docstring.

Reply:
done